### PR TITLE
Document CLI and bundle.json

### PR DESCRIPTION
**Pull request by benLogN**
Hi there,

I was poking around getting into pony and stable looked like a fun place to get my feet wet. I saw you had an issue open to document how to use stable and the bundle.json file format (#19) and figured I'd toss something together for you.

These doc pages can be read as-is, or can be transformed into manpages using [ronn](http://rtomayko.github.io/ronn/) or a few other markdown-to-roff converters. Let me know if you're interested in generating this on travis and getting manpages into your vendored packages.

### CODE REVIEW

**Comment by mfelsche** [comment]
Pretty cool @benLogN !

Me personally i would be very interested in having man-pages alongside the binary in the packages.
But i am also fine with merging these as is.

**Comment by SeanTAllen** [comment]
@mfelsche should this be merged?

File: doc/cli/stable-add.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+stable-add(1) -- add a dependency
+=================================
+
+## SYNOPSIS
+
+    stable add github <url-path> [options]
```
Comment (mfelsche) [inline doc/cli/stable-add.md:6]: we need to add gitlab here, to stay in sync with the code

Comment (mfelsche) [reply doc/cli/stable-add.md:6]: ```stable add gitlab <url-path> [options]```

File: doc/files/stable-config.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+stable-config(5) -- bundle.json file format
```
Comment (mfelsche) [inline doc/files/stable-config.md:1]: this need the renaming to `tack.json`

File: doc/files/stable-config.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+`bundle.json` currently only has one key at the root of the JSON document:
+`deps`. This is an array of dependencies. A minimal `bundle.json` looks like
+
+    { "deps" : [] }
+
+The contents of this array are dependency objects. All dependency objects
+have a `type`; the rest of the fields depend on the type of dependency.
+
+## github
```
Comment (mfelsche) [inline doc/files/stable-config.md:20]: here we also need to add a `gitlab` section

**Comment by mfelsche** [comment]
@benLogN Are you around to do the changes i mentioned above or are you cool with me doing them if you don't have the time?

**Comment by SeanTAllen** [comment]
@mfelsche i dont think any tack related changes should be done until the rename is done. there's a branch for that. i'd say make these changes without the tack related changes then rebase once this is merged and make any tack changes on the tack branch.

**Comment by mfelsche** [comment]
circleci is complaining as this branch was started when we have still been on version 1, which is now deprecated. Travis is fine.
Merging.

**Pull request merged**